kratos/tests/auxiliar_files_for_python_unittest/mdpa_files/test_model_part_io_write.mdpa

```diff
@@ -25,6 +25,10 @@ Begin Elements	Element2D4N
 	3	1	2	1	3	6	
 End Elements
 
+Begin Geometries	Quadrilateral2D4
+	1	4	6	2	3	
+End Geometries
```

Comment:
I'd add another geometry with different type to check that the mixing geometries case also works.

Reply:
Okay

Reply:
I can't, because due to the @tteschemacher the mdpa is reordered each time is created and the tests fails

Reply:
(but it works, just that the mdpa generated is different, we can compare reading again the file instead of just comparing the files)